src/scss/vault.scss

```diff
@@ -1,4 +1,9 @@
-﻿@import "variables.scss";
+﻿/* Cozy UI utilities classes */
+@import '~cozy-ui/dist/cozy-ui.min.css';
+/* Cozy UI React components styles */
+@import '~cozy-ui/transpiled/react/stylesheet.css';
+
+@import "variables.scss";
```

Comment:
Is it really related to `cozy-ui` styles?

Reply:
I'm not sure, this comment is from the original code : https://github.com/cozy/cozy-passwords/blob/master/src/styles/index.css

Should I rewrite it with something like "Utility css classes" ?

Reply:
My bad, I should have been clearer. My question was now about `cozy-ui` importation, but the latest import aka `variables.scss`. Why this import?

Reply:
It's a false positive from the diff tool, this `@import` was already there.